docs/docs.go

```diff
@@ -0,0 +1,265 @@
+// Package docs GENERATED BY SWAG; DO NOT EDIT
```

Comment:
If this file is generated, why does it need to be checked in?  Can't the generation be done by the build?

Reply:
If we do as part of the build, we must run doc generation on for every build otherwise the build will fail.

Reply:
I also think it makes sense to always have the latest docs checked in for API reference without having to do a build to get them.